Content.Client/GameObjects/Components/Clothing/ClothingComponent.cs

```diff
@@ -1,20 +1,23 @@
-﻿#nullable enable
+#nullable enable
 using Content.Client.GameObjects.Components.HUD.Inventory;
 using Content.Client.GameObjects.Components.Items;
 using Content.Shared.GameObjects;
 using Content.Shared.GameObjects.Components.Inventory;
 using Content.Shared.GameObjects.Components.Items;
+using Content.Shared.GameObjects.Components.Storage;
 using Robust.Client.Graphics;
+using Robust.Client.ResourceManagement;
 using Robust.Shared.Containers;
 using Robust.Shared.GameObjects;
+using Robust.Shared.IoC;
 using Robust.Shared.Serialization.Manager.Attributes;
 using Robust.Shared.ViewVariables;
 
 namespace Content.Client.GameObjects.Components.Clothing
 {
     [RegisterComponent]
+    [ComponentReference(typeof(SharedItemComponent))]
     [ComponentReference(typeof(ItemComponent))]
-    [ComponentReference(typeof(IItemComponent))]
     public class ClothingComponent : ItemComponent
```

Comment:
I assume this is getting refactored eventually?

Reply:
Will handle later, was focusing on getting just items into line.